### PR TITLE
sphinx: 5.3.0 -> 6.1.3

### PR DESCRIPTION
**Pull request by ncfavier**
https://www.sphinx-doc.org/en/master/changes.html#release-6-1-3-released-jan-10-2023

Unblocks https://github.com/NixOS/nixpkgs/pull/208594#issuecomment-1380037388

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ncfavier** [comment]
@ofborg build sphinx

**Comment by ncfavier** [comment]
@sternenseemann should we patch GHC with https://gitlab.haskell.org/ghc/ghc/-/commit/00dc51060881df81258ba3b3bdf447294618a4de?

**Comment by mweinelt** [comment]
Removed from python-updates, because it caused breakage, where the upstream wasn't ready. This honestly needs more time.

**Comment by sternenseemann** [comment]
> @sternenseemann should we patch GHC with https://gitlab.haskell.org/ghc/ghc/-/commit/00dc51060881df81258ba3b3bdf447294618a4de?

Yes, can probably be applied before the sphinx update already on `haskell-updates`. Just a bit of an annoyance since it's a full rebuild.

File: pkgs/development/python-modules/sphinx/default.nix [hunk 1]
```diff
@@ -32,33 +28,35 @@
...
 , pytestCheckHook
-, typed-ast
 }:
 
 buildPythonPackage rec {
   pname = "sphinx";
-  version = "5.3.0";
-  format = "pyproject";
+  version = "6.1.3";
+  format = "flit";
```
Comment (mweinelt) [inline pkgs/development/python-modules/sphinx/default.nix:36]:
```suggestion
  format = "pyproject";
```

Please roll that back. We are going to deprecate the `flit` format.

**Comment by mweinelt** [comment]
Please add an anchor to the relevant version in the changelog url.

https://www.sphinx-doc.org/en/master/changes.html#release-6-1-3-released-jan-10-2023

**Comment by mweinelt** [comment]
We will likely have to revert to a version before 6.0, given a serious regression about importing extension modules.

A simpler reproducer is building `python3.pkgs.sphinx-notfound-page` with the changes below. The docs build fails to load `sphinxcontrib-jquery`.

```patch
diff --git a/pkgs/development/python-modules/sphinx-notfound-page/default.nix b/pkgs/development/python-modules/sphinx-notfound-page/default.nix
index dd2efe81fd1..c9fa2a10e63 100644
--- a/pkgs/development/python-modules/sphinx-notfound-page/default.nix
+++ b/pkgs/development/python-modules/sphinx-notfound-page/default.nix
@@ -9,6 +9,7 @@
 , sphinx-rtd-theme
 , sphinx-tabs
 , sphinxcontrib-autoapi
+, sphinxcontrib-jquery
 , sphinxemoji
 # runtime dependencies
 , sphinx
@@ -35,6 +36,7 @@ buildPythonPackage rec {
     sphinx-rtd-theme
     sphinx-tabs
     sphinxcontrib-autoapi
+    sphinxcontrib-jquery
     sphinxemoji
   ];
```

The build log below prints `sys.path` for debug purposes. Another patch against `sphinx` itself.

```patch
diff --git a/sphinx/registry.py b/sphinx/registry.py
index 5a39c3546..b64959c36 100644
--- a/sphinx/registry.py
+++ b/sphinx/registry.py
@@ -442,6 +442,8 @@ class SphinxComponentRegistry:
             try:
                 mod = import_module(extname)
             except ImportError as err:
+                import sys
+                print(sys.path)
                 logger.verbose(__('Original exception:\n') + traceback.format_exc())
                 raise ExtensionError(__('Could not import extension %s') % extname,
                                      err) from err
```



```
sphinx-notfound-page> buildSphinxPhase
sphinx-notfound-page> Executing buildSphinxPhase
sphinx-notfound-page> Sphinx documentation found in docs
sphinx-notfound-page> Executing sphinx-build with html builder
sphinx-notfound-page> Running Sphinx v6.1.3
sphinx-notfound-page> WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid language code. Falling back to 'en' (English).
sphinx-notfound-page> making output directory... done
sphinx-notfound-page> locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exists
sphinx-notfound-page> ['/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/bin', '/nix/store/dx8j6470l3qm2y33wims2ki9d66hn1qn-python3.10-sphinx-notfound-page-0.8.3/lib/python3.10/site-packages', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10/site-packages', '/nix/store/gws8m641iixa2hc4q5bg9njppvxa4ylw-python3.10-flit-3.8.0/lib/python3.10/site-packages', '/nix/store/y6i12fzf6pzmnn8ssdhq19ch26rlmsvi-python3.10-docutils-0.19/lib/python3.10/site-packages', '/nix/store/gyy0i0bblpm1jpim7w0haf6yk1vxlkcm-python3.10-requests-2.28.2/lib/python3.10/site-packages', '/nix/store/x2rdsrmh3ac1j5sahswyvj23lmj3mlx1-python3.10-brotlicffi-1.0.9.2/lib/python3.10/site-packages', '/nix/store/zr5z2qx4fg4hbdc1qr0r88sdc9sy6df4-python3.10-cffi-1.15.1/lib/python3.10/site-packages', '/nix/store/d5y1bx3amihlbdkq9lirc13h1xfhc8j6-python3.10-pycparser-2.21/lib/python3.10/site-packages', '/nix/store/1i3jb4mainr3n91x6a27brcf76v0kswb-python3.10-certifi-2022.12.07/lib/python3.10/site-packages', '/nix/store/g11p0rxwghyqs4q698kfqhv1mrq7habc-python3.10-charset-normalizer-3.0.1/lib/python3.10/site-packages', '/nix/store/81rj50s67rqdvw7hm61fgzzxy9401ybq-python3.10-idna-3.4/lib/python3.10/site-packages', '/nix/store/w7y19mx90yd1yy5hp3hc3f4qnna62vbi-python3.10-urllib3-1.26.14/lib/python3.10/site-packages', '/nix/store/r4m2rbfpzv229d265jx1fvjylc5kv4jx-python3.10-brotli-1.0.9/lib/python3.10/site-packages', '/nix/store/34j7jpvzbbgbi9p37xlhgfnmqxjdv5q5-python3.10-pysocks-1.7.1/lib/python3.10/site-packages', '/nix/store/y9xbvqk8xc27rc3jkaxx4xi6lqj2m4hl-python3.10-flit-core-3.8.0/lib/python3.10/site-packages', '/nix/store/lnjdzml9ymq83fy5sdw64hisdfj5qr0k-python3.10-tomli-w-1.0.0/lib/python3.10/site-packages', '/nix/store/z8kx9f18qnkblqg6i0j4cagrhblzf73j-python3.10-pip-23.0.1/lib/python3.10/site-packages', '/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages', '/nix/store/h3ipn6nsqhcqcgrwiccbwl6szpvb9g73-python3.10-babel-2.11.0/lib/python3.10/site-packages', '/nix/store/vcs8l52vfq2i2kcmdf4spf5d25vp7wym-python3.10-pytz-2022.7.1/lib/python3.10/site-packages', '/nix/store/120wzissg5bdmbg00rzirih5ywi8hbha-python3.10-alabaster-0.7.13/lib/python3.10/site-packages', '/nix/store/2ari54j09y4y1x8cvifa4fzbqf502a28-python3.10-pygments-2.14.0/lib/python3.10/site-packages', '/nix/store/xms99lxb5m8m6dp1p74b2y2i1yhbsr2a-python3.10-imagesize-1.4.1/lib/python3.10/site-packages', '/nix/store/0k18a9kdp0khd67i841jfrwh27w4cdfq-python3.10-Jinja2-3.1.2/lib/python3.10/site-packages', '/nix/store/kxj0zzhx67rrg7vzlpadbljx648liihy-python3.10-markupsafe-2.1.2/lib/python3.10/site-packages', '/nix/store/nivx1kk3syjvy76df35cfmlzkscszz8d-python3.10-packaging-23.0/lib/python3.10/site-packages', '/nix/store/hn9j7l0iaa22nhw849h1n4mvrl5rsall-python3.10-snowballstemmer-2.2.0/lib/python3.10/site-packages', '/nix/store/ia1zgycjgjs954psqzzm6p11x122214d-python3.10-pystemmer-2.2.0/lib/python3.10/site-packages', '/nix/store/m1y2pjak5dxc4w3mimhl1q1zhrj0507w-python3.10-sphinxcontrib-applehelp-1.0.2/lib/python3.10/site-packages', '/nix/store/v20fkpv6pp9zw6ica5h3fapgsr3jdw0k-python3.10-sphinxcontrib-devhelp-1.0.2/lib/python3.10/site-packages', '/nix/store/6qd9hlznw0sdbd25p0kh016jm7vm5m2c-python3.10-sphinxcontrib-htmlhelp-2.0.0/lib/python3.10/site-packages', '/nix/store/wd8bsx5bss80ppb9jkvpbp0zx0ii5mp2-python3.10-sphinxcontrib-jsmath-1.0.1/lib/python3.10/site-packages', '/nix/store/igh8vkm6lx3647s3mlrjdviybv1mcgmw-python3.10-sphinxcontrib-qthelp-1.0.3/lib/python3.10/site-packages', '/nix/store/8kg3z91kyg5wa0cw9h3r4licaxmlih86-python3.10-sphinxcontrib-serializinghtml-1.1.5/lib/python3.10/site-packages', '/nix/store/1x1wqnskyhjwhnshfg4lscp6x52w9adp-python3.10-sphinxcontrib-websupport-1.2.4/lib/python3.10/site-packages', '/nix/store/qh89mdvzma74jjlq1zzqgjzj881q3gz6-python3.10-six-1.16.0/lib/python3.10/site-packages', '/nix/store/j9yl2xrqgh5rlaznmxjxg1si1qswq7x5-python3.10-sphinxcontrib-apidoc-0.3.0/lib/python3.10/site-packages', '/nix/store/zdjlffwqp7dy6q4s9fbyc31hdq9ic01j-python3.10-pbr-5.11.1/lib/python3.10/site-packages', '/nix/store/9z2cm958bxarf7n5i48czm7p7lwf9s58-python3.10-setuptools-67.4.0/lib/python3.10/site-packages', '/nix/store/gya0siaayqd8kmih8zpv60n7kky6gg5a-python3.10-sphinx-prompt-1.6.0/lib/python3.10/site-packages', '/nix/store/gxxyzvnhgr2wpx3zcybpdkxza6qdbz4d-python3.10-sphinx-rtd-theme-1.2.0/lib/python3.10/site-packages', '/nix/store/zzf38chhrx6x3yl2bcc9v529p3msiv2d-python3.10-sphinxcontrib-jquery-3.0.0/lib/python3.10/site-packages', '/nix/store/qcbcw9l02hxamn4si7a3jcpbczw3w16h-python3.10-sphinx-tabs-3.4.1/lib/python3.10/site-packages', '/nix/store/bi45zsw4plcbjcyxkna9jizngqj1rl6h-python3.10-sphinx-autoapi-2.0.1/lib/python3.10/site-packages', '/nix/store/vp3nh4r1sfpp643r4kv8505r93gdx51q-python3.10-astroid-2.14.2/lib/python3.10/site-packages', '/nix/store/2pf2w3wjczj7p9zij92pld15c0cbrwaf-python3.10-lazy-object-proxy-1.9.0/lib/python3.10/site-packages', '/nix/store/ir5blszgk6sncai8104n2xs26civ6zy0-python3.10-wrapt-1.14.1/lib/python3.10/site-packages', '/nix/store/i5kf4xa9x6rzwxp2kvx1z2pl57jf6snh-python3.10-pyyaml-6.0/lib/python3.10/site-packages', '/nix/store/j7lsb6q0j269f3n1gl08nbq0h33girlr-python3.10-unidecode-1.3.6/lib/python3.10/site-packages', '/nix/store/wh3gi2wbbxii3b73s9vir7p4sj8g3h9w-python3.10-typing-extensions-4.5.0/lib/python3.10/site-packages', '/nix/store/vn8zghk5c18gc4mwbw493ifnrlycr9y1-python3.10-sphinxemoji-0.2.0/lib/python3.10/site-packages', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python310.zip', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10/lib-dynload']
sphinx-notfound-page> Original exception:
sphinx-notfound-page> Traceback (most recent call last):
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/registry.py", line 443, in load_extension
sphinx-notfound-page>     mod = import_module(extname)
sphinx-notfound-page>   File "/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10/importlib/__init__.py", line 126, in import_module
sphinx-notfound-page>     return _bootstrap._gcd_import(name[level:], package, level)
sphinx-notfound-page>   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
sphinx-notfound-page>   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
sphinx-notfound-page>   File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
sphinx-notfound-page> ModuleNotFoundError: No module named 'sphinxcontrib.jquery'
sphinx-notfound-page> 
sphinx-notfound-page> 
sphinx-notfound-page> Traceback (most recent call last):
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/registry.py", line 443, in load_extension
sphinx-notfound-page>     mod = import_module(extname)
sphinx-notfound-page>   File "/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10/importlib/__init__.py", line 126, in import_module
sphinx-notfound-page>     return _bootstrap._gcd_import(name[level:], package, level)
sphinx-notfound-page>   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
sphinx-notfound-page>   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
sphinx-notfound-page>   File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
sphinx-notfound-page> ModuleNotFoundError: No module named 'sphinxcontrib.jquery'
sphinx-notfound-page> 
sphinx-notfound-page> The above exception was the direct cause of the following exception:
sphinx-notfound-page> 
sphinx-notfound-page> Traceback (most recent call last):
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/cmd/build.py", line 279, in build_main
sphinx-notfound-page>     app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/application.py", line 262, in __init__
sphinx-notfound-page>     self._init_builder()
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/application.py", line 334, in _init_builder
sphinx-notfound-page>     self.builder.init()
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/builders/html/__init__.py", line 254, in init
sphinx-notfound-page>     self.init_templates()
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/builders/html/__init__.py", line 305, in init_templates
sphinx-notfound-page>     self.theme = theme_factory.create(themename)
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/theming.py", line 225, in create
sphinx-notfound-page>     self.load_extra_theme(name)
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/theming.py", line 179, in load_extra_theme
sphinx-notfound-page>     self.load_external_theme(name)
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/theming.py", line 195, in load_external_theme
sphinx-notfound-page>     self.app.registry.load_extension(self.app, entry_point.module)
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/registry.py", line 458, in load_extension
sphinx-notfound-page>     metadata = setup(app)
sphinx-notfound-page>   File "/nix/store/gxxyzvnhgr2wpx3zcybpdkxza6qdbz4d-python3.10-sphinx-rtd-theme-1.2.0/lib/python3.10/site-packages/sphinx_rtd_theme/__init__.py", line 60, in setup
sphinx-notfound-page>     app.setup_extension("sphinxcontrib.jquery")
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/application.py", line 398, in setup_extension
sphinx-notfound-page>     self.registry.load_extension(self, extname)
sphinx-notfound-page>   File "/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages/sphinx/registry.py", line 448, in load_extension
sphinx-notfound-page>     raise ExtensionError(__('Could not import extension %s') % extname,
sphinx-notfound-page> sphinx.errors.ExtensionError: Could not import extension sphinxcontrib.jquery (exception: No module named 'sphinxcontrib.jquery')
sphinx-notfound-page> 
sphinx-notfound-page> Extension error:
sphinx-notfound-page> Could not import extension sphinxcontrib.jquery (exception: No module named 'sphinxcontrib.jquery')
```

When taking that `sys.path` into a python interpreter, importing works just fine.

```
❯ python3
Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.path = ['/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/bin', '/nix/store/dx8j6470l3qm2y33wims2ki9d66hn1qn-python3.10-sphinx-notfound-page-0.8.3/lib/python3.10/site-packages', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10/site-packages', '/nix/store/gws8m641iixa2hc4q5bg9njppvxa4ylw-python3.10-flit-3.8.0/lib/python3.10/site-packages', '/nix/store/y6i12fzf6pzmnn8ssdhq19ch26rlmsvi-python3.10-docutils-0.19/lib/python3.10/site-packages', '/nix/store/gyy0i0bblpm1jpim7w0haf6yk1vxlkcm-python3.10-requests-2.28.2/lib/python3.10/site-packages', '/nix/store/x2rdsrmh3ac1j5sahswyvj23lmj3mlx1-python3.10-brotlicffi-1.0.9.2/lib/python3.10/site-packages', '/nix/store/zr5z2qx4fg4hbdc1qr0r88sdc9sy6df4-python3.10-cffi-1.15.1/lib/python3.10/site-packages', '/nix/store/d5y1bx3amihlbdkq9lirc13h1xfhc8j6-python3.10-pycparser-2.21/lib/python3.10/site-packages', '/nix/store/1i3jb4mainr3n91x6a27brcf76v0kswb-python3.10-certifi-2022.12.07/lib/python3.10/site-packages', '/nix/store/g11p0rxwghyqs4q698kfqhv1mrq7habc-python3.10-charset-normalizer-3.0.1/lib/python3.10/site-packages', '/nix/store/81rj50s67rqdvw7hm61fgzzxy9401ybq-python3.10-idna-3.4/lib/python3.10/site-packages', '/nix/store/w7y19mx90yd1yy5hp3hc3f4qnna62vbi-python3.10-urllib3-1.26.14/lib/python3.10/site-packages', '/nix/store/r4m2rbfpzv229d265jx1fvjylc5kv4jx-python3.10-brotli-1.0.9/lib/python3.10/site-packages', '/nix/store/34j7jpvzbbgbi9p37xlhgfnmqxjdv5q5-python3.10-pysocks-1.7.1/lib/python3.10/site-packages', '/nix/store/y9xbvqk8xc27rc3jkaxx4xi6lqj2m4hl-python3.10-flit-core-3.8.0/lib/python3.10/site-packages', '/nix/store/lnjdzml9ymq83fy5sdw64hisdfj5qr0k-python3.10-tomli-w-1.0.0/lib/python3.10/site-packages', '/nix/store/z8kx9f18qnkblqg6i0j4cagrhblzf73j-python3.10-pip-23.0.1/lib/python3.10/site-packages', '/nix/store/ak26q3w7d48fja5dmw3z6w3z5ms2wn5w-python3.10-sphinx-6.1.3/lib/python3.10/site-packages', '/nix/store/h3ipn6nsqhcqcgrwiccbwl6szpvb9g73-python3.10-babel-2.11.0/lib/python3.10/site-packages', '/nix/store/vcs8l52vfq2i2kcmdf4spf5d25vp7wym-python3.10-pytz-2022.7.1/lib/python3.10/site-packages', '/nix/store/120wzissg5bdmbg00rzirih5ywi8hbha-python3.10-alabaster-0.7.13/lib/python3.10/site-packages', '/nix/store/2ari54j09y4y1x8cvifa4fzbqf502a28-python3.10-pygments-2.14.0/lib/python3.10/site-packages', '/nix/store/xms99lxb5m8m6dp1p74b2y2i1yhbsr2a-python3.10-imagesize-1.4.1/lib/python3.10/site-packages', '/nix/store/0k18a9kdp0khd67i841jfrwh27w4cdfq-python3.10-Jinja2-3.1.2/lib/python3.10/site-packages', '/nix/store/kxj0zzhx67rrg7vzlpadbljx648liihy-python3.10-markupsafe-2.1.2/lib/python3.10/site-packages', '/nix/store/nivx1kk3syjvy76df35cfmlzkscszz8d-python3.10-packaging-23.0/lib/python3.10/site-packages', '/nix/store/hn9j7l0iaa22nhw849h1n4mvrl5rsall-python3.10-snowballstemmer-2.2.0/lib/python3.10/site-packages', '/nix/store/ia1zgycjgjs954psqzzm6p11x122214d-python3.10-pystemmer-2.2.0/lib/python3.10/site-packages', '/nix/store/m1y2pjak5dxc4w3mimhl1q1zhrj0507w-python3.10-sphinxcontrib-applehelp-1.0.2/lib/python3.10/site-packages', '/nix/store/v20fkpv6pp9zw6ica5h3fapgsr3jdw0k-python3.10-sphinxcontrib-devhelp-1.0.2/lib/python3.10/site-packages', '/nix/store/6qd9hlznw0sdbd25p0kh016jm7vm5m2c-python3.10-sphinxcontrib-htmlhelp-2.0.0/lib/python3.10/site-packages', '/nix/store/wd8bsx5bss80ppb9jkvpbp0zx0ii5mp2-python3.10-sphinxcontrib-jsmath-1.0.1/lib/python3.10/site-packages', '/nix/store/igh8vkm6lx3647s3mlrjdviybv1mcgmw-python3.10-sphinxcontrib-qthelp-1.0.3/lib/python3.10/site-packages', '/nix/store/8kg3z91kyg5wa0cw9h3r4licaxmlih86-python3.10-sphinxcontrib-serializinghtml-1.1.5/lib/python3.10/site-packages', '/nix/store/1x1wqnskyhjwhnshfg4lscp6x52w9adp-python3.10-sphinxcontrib-websupport-1.2.4/lib/python3.10/site-packages', '/nix/store/qh89mdvzma74jjlq1zzqgjzj881q3gz6-python3.10-six-1.16.0/lib/python3.10/site-packages', '/nix/store/j9yl2xrqgh5rlaznmxjxg1si1qswq7x5-python3.10-sphinxcontrib-apidoc-0.3.0/lib/python3.10/site-packages', '/nix/store/zdjlffwqp7dy6q4s9fbyc31hdq9ic01j-python3.10-pbr-5.11.1/lib/python3.10/site-packages', '/nix/store/9z2cm958bxarf7n5i48czm7p7lwf9s58-python3.10-setuptools-67.4.0/lib/python3.10/site-packages', '/nix/store/gya0siaayqd8kmih8zpv60n7kky6gg5a-python3.10-sphinx-prompt-1.6.0/lib/python3.10/site-packages', '/nix/store/gxxyzvnhgr2wpx3zcybpdkxza6qdbz4d-python3.10-sphinx-rtd-theme-1.2.0/lib/python3.10/site-packages', '/nix/store/zzf38chhrx6x3yl2bcc9v529p3msiv2d-python3.10-sphinxcontrib-jquery-3.0.0/lib/python3.10/site-packages', '/nix/store/qcbcw9l02hxamn4si7a3jcpbczw3w16h-python3.10-sphinx-tabs-3.4.1/lib/python3.10/site-packages', '/nix/store/bi45zsw4plcbjcyxkna9jizngqj1rl6h-python3.10-sphinx-autoapi-2.0.1/lib/python3.10/site-packages', '/nix/store/vp3nh4r1sfpp643r4kv8505r93gdx51q-python3.10-astroid-2.14.2/lib/python3.10/site-packages', '/nix/store/2pf2w3wjczj7p9zij92pld15c0cbrwaf-python3.10-lazy-object-proxy-1.9.0/lib/python3.10/site-packages', '/nix/store/ir5blszgk6sncai8104n2xs26civ6zy0-python3.10-wrapt-1.14.1/lib/python3.10/site-packages', '/nix/store/i5kf4xa9x6rzwxp2kvx1z2pl57jf6snh-python3.10-pyyaml-6.0/lib/python3.10/site-packages', '/nix/store/j7lsb6q0j269f3n1gl08nbq0h33girlr-python3.10-unidecode-1.3.6/lib/python3.10/site-packages', '/nix/store/wh3gi2wbbxii3b73s9vir7p4sj8g3h9w-python3.10-typing-extensions-4.5.0/lib/python3.10/site-packages', '/nix/store/vn8zghk5c18gc4mwbw493ifnrlycr9y1-python3.10-sphinxemoji-0.2.0/lib/python3.10/site-packages', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python310.zip', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10', '/nix/store/grmc7s8xvc6khbgs0g0rkssc883wwncx-python3-3.10.10/lib/python3.10/lib-dynload']
>>> from importlib import import_module
>>> import_module("sphinxcontrib.jquery")
<module 'sphinxcontrib.jquery' from '/nix/store/zzf38chhrx6x3yl2bcc9v529p3msiv2d-python3.10-sphinxcontrib-jquery-3.0.0/lib/python3.10/site-packages/sphinxcontrib/jquery/__init__.py'>
```

I'm out of ideas at this point.

**Comment by ncfavier** [comment]
Is [this](https://blog.readthedocs.com/sphinx6-upgrade/) relevant?

> [sphinxcontrib-jquery](https://pypi.org/project/sphinxcontrib.jquery/) 3.0.0 does not work on local builds while version 2.0.0 relies on Google’s CDN. [A fix](https://github.com/sphinx-contrib/jquery/pull/14) is waiting for review and release.

**Comment by mweinelt** [comment]
No, our failure happens while trying to import sphinxcontrib-jquery from within sphinx.

**Pull request merged**